integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -177,8 +181,70 @@ If you want to experiment with different prompts, do the following:
+| Invoke GPT Chat Completion API | Connects to OpenAI and, using as context the documents resulting from the search, asks the language model to generate an answer to the user query. It is not meant to be invoked directly, but rather by the "Generate Answer" action.                                                                          |
+
+To use the sample actions:
+
+1. **After having configured both extensions**, download the sample actions from this starter kit: [`discovery-openai-actions.json`](./discovery-openai-actions.json).
+
+1. Use **Actions Global Settings** to upload the JSON file to your assistant. For more information, see [Uploading](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import).
+
+1. Under "Variables"/"Created by you" (within the Actions page), set the `discovery_project_id` session variable using the project ID value you obtained [when configuring Watson Discovery above](#configure-watson-discovery-extension).
+
+**NOTE**: If you import the actions _before_ configuring the extensions, you will see some errors on the actions because it could not find the extensions. Simply configure the extensions as described above and re-import the action JSON file.
+
+#### Session variables
+
+These are the session variables used in this example. Several of them are automatically set in the sample [discovery-openai-actions.json](discovery-openai-actions.json). You will need to set the `discovery_project_id` to point to the project id for your Watson Discovery collection.
+
+- `discovery_date_version` - Discovery date versions are documented in the [release notes](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-release-notes).
+- `discovery_project_id`: You **MUST** set this value to be the project ID obtained [when configuring Watson Discovery above](#configure-watson-discovery-extension).
+- `messages` : Input to the OpenAI model; includes the `search_results` and the `model_prompt`.
+- `model_for_chat` : The OpenAI model used, defaults to `gpt-3.5-turbo`.
+- `model_max_tokens` : The maximum number of text fragments to input to the model. The starter kit uses 200.
+- `model_prompt`: You MAY change this to do prompt engineering, but a default will be used by the model if you don’t pass a prompt here.
+- `model_response`: The text generated by the model in response to the `messages`.
+- `passages` : Concatenation of top search results.
+- `query_text`: You MAY change this to pass queries to Watson Discovery. By default the Search action passes the user’s input.text directly.
+- `search_results`: Response object from [Discovery query](https://cloud.ibm.com/apidocs/discovery-data#query).
+- `snippet` : Top results from the Watson Discovery document search.
```

Comment:
I saw that you have mentioned one required param which is project_id, but specifying that these session variables need not be set and are optional can reduce some confusion.

Reply:
The README says that most of these variables are automatically set in the actions file. That is not the same as saying that they are optional or don't need to be set. I will clarify that text.